modules/page-templates/templates/canvas.php

```diff
@@ -23,6 +23,7 @@
 </head>
 <body <?php body_class(); ?>>
 	<?php
+	Elementor\Modules\PageTemplates\Module::body_open();
```

Comment:
Why not add here `do_action( 'wp_body_open' );`?

Reply:
When we drop all `WP < 5.2`  support, we just remove that method.

Reply:
With `do_action( 'wp_body_open' )` we shouldn't do anything.